api-watch.opam

```diff
@@ -13,6 +12,10 @@ depends: [
   "logs"
   "cmdliner" {>= "1.1.0"}
   "odoc" {with-doc}
+  "diffutils"
+]
+pin-depends: [
```

Comment:
I forgot to mention it but our opam file is generated by dune so we don't edit it directly. See https://dune.readthedocs.io/en/latest/howto/opam-file-generation.html for opam generation documentation.

You should instead edit the `dune-project` file. You'll be able to add diffutils to the list of dependencies but you won't be able to add a pin-depends there. To add pin-depends we'll instead have to add an `api-watch.opam.template` file to the repo and set the fields that can't be set in `package` stanza of our `dune-project` there.